### PR TITLE
NO-ISSUE: Fixing typo in RegisteredServiceThemeBasedViewResolver bean def

### PR DESCRIPTION
**Pull request by jtgasper3**

### CODE REVIEW

**Comment by jtgasper3** [comment]
Hold off on this PR. More changes are needed.

**Comment by jtgasper3** [comment]
All better. This is the working combination.

File: cas-server-documentation/installation/User-Interface-Customization.md [hunk 1]
```diff
@@ -364,15 +364,19 @@ associated theme to selectively choose which set of UI views will be used to gen
 for a different type of audience are entirely different structurally that simply
 using the `ServiceThemeResolver` is not practical to augment the default views. In such cases, new view pages may be required.
 
-Views associated with a particular theme by default are expected to be found at: `/WEB-INF/view/jsp/<theme-id>/ui/` 
+Replace the `internalViewResolver` definition with the following:
 
 {% highlight xml %}
-<bean id="themeResolver" class="org.jasig.cas.services.web.RegisteredServiceThemeBasedViewResolver"
-    c:defaultThemeName="${cas.themeResolver.defaultThemeName}"
+<bean id="internalViewResolver" class="org.jasig.cas.services.web.RegisteredServiceThemeBasedViewResolver"
```
Comment (mmoayyed) [inline cas-server-documentation/installation/User-Interface-Customization.md]: Why change the bean name?

See http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/servlet/DispatcherServlet.html

**Comment by jtgasper3** [comment]
Because us implementors are lazy and using a different name would require a change to https://github.com/Jasig/cas/blob/4.1.x/cas-server-webapp/src/main/webapp/WEB-INF/cas-servlet.xml#L251 as well. (That list seems to be chained back to the `loginFlowRegistry`, so a change seem necessary.)

Also, if I recall the RegisteredServiceThemeBasedViewResolver is an InternalViewResolver, so technically the name still fits. :)

**Comment by mmoayyed** [comment]
Very cool. I was under the assumption that the name of the bean shall remain themeResolver for Spring to pick it up, but this obviously works too. Thanks!

**Comment by jtgasper3** [comment]
ThemeResolver is wrong. This is a ViewResolver.. :)

**Comment by mmoayyed** [comment]
Of course. What was I thinking!

**Pull request merged**